homeassistant/components/zha/core/const.py

```diff
@@ -149,16 +156,51 @@
 class RadioType(enum.Enum):
     """Possible options for radio type."""
 
-    ezsp = "ezsp"
-    deconz = "deconz"
-    ti_cc = "ti_cc"
-    zigate = "zigate"
-    xbee = "xbee"
+    ezsp = (
+        "ESZP: Radio based on Silabs chip using EmberZNet protocol",
```

Comment:
Note that this text won't be translatable. The names will probably be understood, but the more English we add the more English knowledge we require from the user.

Reply:
Hrm, i see your point. But on the other hand  we're not translating those currently anyway, do we? 
My assumptions:
- most of the time if user pick the right com port, the radio would be auto detected
- If they get here, they will be able to see a "familiar name" like Conbee???
Means i have to add HUSBZB-1 and Telegesis to EZSP choice

Reply:
Added more names and less English since this won't be translated